docs/en/ingest-management/release-notes/release-notes-8.17.asciidoc

```diff
@@ -40,6 +41,34 @@ Review important information about the 8.17.7 release.
 
 * Upgrade Go version to v1.24.3 in {{agent}}. {agent-pull}8109[https://github.com/elastic/elastic-agent/pull/8109]
 
+[discrete]
+[[known-issues-8.17.7]]
+=== Known issues
+
+[[known-issue-1800-8-17-7]]
+.On Windows, {agent} is unable to enroll into {fleet} following the {agent}'s upgrade.
+[%collapsible]
+====
+
+*Details* +
+There is a known issue where an {agent} installed on Windows and previously enrolled into {fleet} is unable to re-enroll after the {agent} is upgraded. Attempting to enroll the {agent} fails with the following error:
```

Comment:
```suggestion
There is a known issue where an {agent} installed on Windows and previously enrolled into {fleet} is unable to re-enroll after the {agent}. Attempting to enroll the {agent} fails with the following error:
```

I think the bug affects new installations as well so let's remove the wording about upgrade to avoid confusion.